lib/analyzers/exercise_analyzer.rb

```diff
@@ -28,6 +28,39 @@ def results
     }
   end
 
+  def approve_if_whitespace_is_sensible(msg = nil, params = {})
+    if solution.indentation_is_sensible?
+      self.comments << if params.any?
+                         { comment: msg, params: }
+                       else
+                         msg
+                       end
+      self.comments.compact!
+      self.status = :approve
+
+      raise FinishedFlowControlException
+    else
+      disapprove("ruby.general.incorrect_indentation")
+    end
+  end
+
+  def disapprove(msg, params = {})
```

Comment:
Here, as mentioned, is the second place the `msg` happens when it is likely `comment`.